logicrunner/goplugin/preprocessor/main.go

```diff
@@ -361,6 +364,12 @@ func (pf *ParsedFile) generateImports(wrapper bool) map[string]bool {
 			extendImportsMap(pf, fun.Type.Results, imports)
 		}
 	}
+
+	if imports[fmt.Sprintf(`"%s"`, foundationPath)] && removeFoundation {
+		delete(imports, fmt.Sprintf(`"%s"`, foundationPath))
+		fmt.Println("WARNING: build proxy without methods")
```

Comment:
please move it closer to methods parsing. 
We do not know anything about methods here